### ISSUE TITLE
VS editor exception

### ISSUE THREAD

**Issue by cpyrgas** [X-Sharp/XSharpPublic#78]
Got the following exception in VS 2015 (not 17). It is with the VS integration version that comes with RC2 build.

System.NullReferenceException: Object reference not set to an instance of an object.
       at XSharp.Project.Editors.BraceMatching.BraceMatchingTagger.&lt;GetTags&gt;d__20.MoveNext() in C:\XSharp\DevPublic\VisualStudio\ProjectPackage\Editors\BraceMatching\BraceMatching.cs:line 170
       at Microsoft.VisualStudio.Text.Tagging.Implementation.TagAggregator`1.&lt;GetTagsForBuffer&gt;d__38.MoveNext()

**Comment by RobertvanderHulst** [comment]
This is strange. The method implements an enumerator for tags. The line of code does a yield break and should simply end the iterator.

**Comment by cpyrgas** [comment]
Today I did get that error multiple times while I was investigated a report. It was always in VS2015 and sometimes I got the exception dialog, other times VS2015 crashed with no message, but I suspect it's the same root problem.

Most of the times the error happened while aggressively pressing (holding down for several seconds) CTRL+Z or CTRL+SHIFT+Z, to undo/redo multiple (dozens) of previous code edits and the callstack is always the same. Could it be that .cs file has been changed since the RC2 release, so the error message is not pointing to the wrong line?
